foundation-ui/utils/ReactUtil.js

```diff
@@ -15,6 +15,10 @@ const ReactUil = {
       return null;
     }
 
+    if (Array.isArray(elements) && elements.length === 1 && !alwaysWrap) {
```

Comment:
I have the feeling that we, with this addition, should change the procedure to avoid unnecessary checks, but that shouldn't block this PR.